mlir-sycl/include/mlir/Dialect/SYCL/IR/SYCLOps.td

```diff
@@ -207,11 +208,15 @@ def SYCL_GroupType : SYCL_Type<"Group", "group"> {
   let assemblyFormat = "`<` `[` $dimension `]` `,` `(` $body `)` `>`";
 }
 
+def VectorEltTy : AnyTypeOf<[I1, I8, I16, I32, I64, F16, F32, F64]>;
```

Comment:
`AnyIntOrFloatTy` would be misleading to me, as there are types we do not allow in vectors, e.g., `I128`. I think this is fine as it describes exactly what it is: the type comprising the types that can be used as vector element types. Also, in case we allow types other than floating point or integral types in vectors in the future, we would just need to add them here.

**Note**: This won't be visible outside this `.td` file. It's just an alias to be used in this file.

Reply:
I missed the I128. Ok I see your point.